src/app/amsterdam/contact/vraag/page.tsx

```diff
@@ -20,32 +31,30 @@ function Question() {
       <Grid.Cell span={{ narrow: 4, medium: 6, wide: 8 }} start={{ narrow: 1, medium: 2, wide: 3 }}>
         <Breadcrumb>
           <NextLink legacyBehavior passHref href="/amsterdam">
-            <Breadcrumb.Item href="/amsterdam">Home</Breadcrumb.Item>
+            <Breadcrumb.Link href="/amsterdam">Home</Breadcrumb.Link>
           </NextLink>
         </Breadcrumb>
         <form className="ams-gap--md" onSubmit={handleSubmit}>
           <Heading>Contact</Heading>
-          <div>
+          <Field>
             <Label htmlFor="description">Wat wilt u aan de gemeente vragen?</Label>
             <Paragraph size="small">
```

Comment:
This should be connected to the TextArea using `aria-describedby`

Reply:
Yes, done.